docs/api/web-contents.md

```diff
@@ -1452,12 +1452,22 @@ Returns `Boolean` - If *offscreen rendering* is enabled returns whether it is cu
 * `fps` Integer
 
 If *offscreen rendering* is enabled sets the frame rate to the specified number.
-Only values between 1 and 60 are accepted.
+Only values between 1 and 240 are accepted.
```

Comment:
Where is the constant `240` defined?

Reply:
That was [bumped from 60 in an earlier PR](https://github.com/electron/electron/blob/c320da5ec80dc74d359216e12b9016bd7e5ce7c0/atom/browser/osr/osr_render_widget_host_view.cc#L1150), I think I'm gonna break this out to a mini docs update PR.